docs/BuildingAndTestingDXC.rst

```diff
@@ -33,9 +33,10 @@ for the configuration process.
 All of the most basic CMake configurations for DXC follow a similar format to:
 
 .. code:: sh
-  cmake <Repository Root> \
-    -C <Repository Root>/cmake/caches/PredefinedParams.cmake \
-    -DCMAKE_BUILD_TYPE=<Build Type> \
+
+  cmake <Repository Root>
```

Comment:
Instead of removing the `\` can you make them `\\` so they're escaped properly? The `\` needs to be preserved in order for the command lines to be able to be copy and pasted into a terminal.

Reply:
on it 👌